src/status_im2/constants.cljs

```diff
@@ -273,3 +273,9 @@
 (def ^:const theme-type-light 1)
 (def ^:const theme-type-dark 2)
 (def ^:const bottom-sheet-animation-delay 450)
+
+(def ^:const local-pair-event-process-success "process-success")
```

Comment:
Hi, where these constants are being used

Reply:
i used it in [handle-local-pairing-signals](https://github.com/status-im/status-react/blob/dd1ee4bedbe6e4a7a1ac46c40db54c45bc456255/src/status_im/signals/core.cljs#L56) before, but now i'd say maybe it will be used in the future, cause local pairing may take a little long, we may use it as progress info? @Parveshdhull